pkg/test/resourcefixture/testdata/basic/compute/v1alpha1/computemanagedsslcertificate/managedsslcertificate/_http.log

```diff
@@ -64,7 +64,7 @@ X-Xss-Protection: 0
   "selfLink": "https://www.googleapis.com/compute/beta/projects/${projectId}/global/operations/${operationID}",
   "startTime": "2024-04-01T12:34:56.123456Z",
   "status": "RUNNING",
-  "targetId": "7780111738275460050",
+  "targetId": "1719006515851113983",
```

Comment:
Do we need a rebase for this one?  (Hopefully this fails tests also, so it's a good check, though it would be sort of interesting if it didn't!)

Reply:
Yes, was waiting for https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/2109/files (now it's merged) when I last updated this PR.

Reply:
> Do we need a rebase for this one? (Hopefully this fails tests also, so it's a good check, though it would be sort of interesting if it didn't!)

BTW, yes it did fail the PRESUBMIT check https://github.com/GoogleCloudPlatform/k8s-config-connector/actions/runs/9626625776/job/26552788995